centreon/www/front_src/src/Resources/Actions/Resource/Check/Text.tsx

```diff
@@ -0,0 +1,51 @@
+import { makeStyles } from 'tss-react/mui';
+import { useTranslation } from 'react-i18next';
+import { equals } from 'ramda';
+
+import { Typography } from '@mui/material';
+
+const useStyles = makeStyles()((theme) => ({
+  description: {
+    fontSize: '0.8rem',
+    whiteSpace: 'pre-line'
+  },
+  title: {
+    color: theme.palette.info.main,
+    fontWeight: 'bold'
+  }
+}));
+
+interface Props {
+  description: string;
+  title: string;
+}
+
+const Text = ({ title, description }: Props): JSX.Element => {
+  const { classes } = useStyles();
+  const { t } = useTranslation();
+
+  const formatLabel = (label: string): string => {
+    return label.charAt(0).toUpperCase() + label.slice(1).toLowerCase();
+  };
+
+  const formatDescription = (data: string): string => {
+    const words = data.split(' ');
+
+    const wordsWithLineBreak = words.map((word, index) => {
+      return equals(index, Math.round(words.length / 2)) ? `${word}\n` : word;
+    });
+
+    return wordsWithLineBreak.join(' ');
+  };
```

Comment:
those are pure functions, and can be moved outside of the rendering cycle (not in the component, declared in the upper scope)

Reply:
a pure function is a function that will always return the same result given the same arguments. 
Your functions here do not rely on any props, or internal state. 
You can move them outside of the react component, if you decide to keep them. 
But I don't think you need them. 

for the `formatLabel` function : 
- why the label is not correctly capitalized to begin with, can we not create a correctly capitalized label within the translatedLabel file ?
- if there is a good reason, using the CSS property `text-transform: capitalize;` makes more sense 


for the `formatDescription` function : 
- why are you trying to split the line in two ? The correct responsive way to handle this is just to have a `max-width` css property that will naturally split the line.

Reply:
for formatDescription  : what value should I put to the max witdh , knowing that we have different languages

Reply:
you did 3 feedback for the same thing  , so i keep just one this